src/renderer/contexts/ExecutionContext.tsx

```diff
@@ -227,65 +227,45 @@ export const ExecutionProvider = memo(({ children }: React.PropsWithChildren<{}>
 
     const [eventSource, eventSourceStatus] = useBackendEventSource(port);
 
-    useBackendEventSourceListener(
-        eventSource,
-        'finish',
-        () => {
-            setStatus(ExecutionStatus.READY);
-        },
-        [
-            // TODO: This is a hack due to useEventSource having a bug related to useEffect jank
-            {},
-            // status isn't actually used
-            status,
-            setStatus,
-        ]
-    );
+    useBackendEventSourceListener(eventSource, 'finish', () => setStatus(ExecutionStatus.READY));
 
-    useBackendEventSourceListener(
-        eventSource,
-        'execution-error',
-        (data) => {
-            if (data) {
-                sendAlert(AlertType.ERROR, null, getExecutionErrorMessage(data, schemata));
-                unAnimate();
-                setStatus(ExecutionStatus.READY);
-            }
-        },
-        [setStatus, unAnimate, schemata, {}]
```

Comment:
Are you 100% sure you didn't cause a whole bunch of bugs by getting rid of these {} s?